app/javascript/src/components/editor/EditorList.svelte

```diff
@@ -25,6 +25,9 @@
       swapTreshhold: 0.25,
       multiDrag: true,
       multiDragKey: "ctrl",
+      fallbackTolerance: 3,
+      delayOnTouchOnly: true,
+      delay: 100,
```

Comment:
Hmm, for some reason moving the selected item makes it jump straight to the bottom. If you select another item, it fixes itself. Tested with device emulation both on and off:

https://github.com/Mitcheljager/workshop.codes/assets/6181929/a00caa67-3888-4a41-a94a-88e65df70851

I have a feeling it is a bug with sortablejs though 🤔

Reply:
Weird 😮 i just tested it on ws.c and its the same thing so it doesnt appear to be caused by this change. 
although im not experiencing any issues without device emulation, only with mobile (same thing after applying these changes, so it seems unrelated).

Reply:
Hmm. I tested it on prod and also in the master branch and it wasn't happening. Then I guess it's a problem for another day.

Reply:
I used Brave, which is Chromium-based. Both times on PC.

Regardless, I've made a ticket for this #432, so we can continue the conversation there.